ui/core/components/character_stats.tsx

```diff
@@ -45,7 +46,7 @@ export class CharacterStats extends Component {
 			const row = (
 			<tr
 				className='character-stats-table-row'
-			>	
+			>
```

Comment:
My editor auto-trims trailing whitespace - there's a few diff lines of that here